### ISSUE TITLE
Tensorflow build broken for Tensorflow r0.12 with Bazel 0.4.4.

### ISSUE THREAD

**Issue by tmaila** [tensorflow/rust#55]
The head release of Rust-Tensorflow links to a release r0.12 of TensorFlow which fails to build with Bazel 0.4.4. This results in Rust-TensorFlow crate build failing. The TensorFlow build tries to fetch broken android models during ./configure step. This configure fails and the whole TensorFlow build fails. 

See explanation here:
https://github.com/bazelbuild/bazel/issues/2478

The same problem can be reproduced by just checking out release r0.12 of TensorFlow and trying to build it using Bazel 0.4.4. 

Linking to a newer release of TensorFlow is likely to solve the problem as manually building TensorFlow r1.0.0 didn't have the same issue.

Crate build output (tail):
```
____Loading package: @inception5h//
ERROR: /home/tomi/.cache/bazel/_bazel_tomi/513f25fe4a2103b04ca570fb64b12a11/external/bazel_tools/src/tools/android/java/com/google/devtools/build/android/dexer/BUILD:3:1: no such target '//external:android/dx_jar_import': target 'android/dx_jar_import' not declared in package 'external' defined by /home/tomi/projects/rust/rust-tensorflow/tensorflow-sys/target/source-v0.12.0/WORKSPACE and referenced by '@bazel_tools//src/tools/android/java/com/google/devtools/build/android/dexer:dexer'.
ERROR: Evaluation of query "deps(//tensorflow/...)" failed: errors were encountered while computing transitive closure.
thread 'main' panicked at 'failed to execute "bash" "-c" "yes \'\'|./configure"', tensorflow-sys/build.rs:105
stack backtrace:
   1:     0x55d015f5129a - std::sys::imp::backtrace::tracing::imp::write::h3188f035833a2635
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:42
   2:     0x55d015f56cef - std::panicking::default_hook::{{closure}}::h6385b6959a2dd25b
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:349
   3:     0x55d015f568ee - std::panicking::default_hook::he4f3b61755d7fa95
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:365
   4:     0x55d015f57137 - std::panicking::rust_panic_with_hook::hf00b8130f73095ec
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:553
   5:     0x55d015f56f74 - std::panicking::begin_panic::h6227f62cb2cdaeb4
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:515
   6:     0x55d015f56ee9 - std::panicking::begin_panic_fmt::h173eadd80ae64bec
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:499
   7:     0x55d015df4e9e - build_script_build::run::ha59c337a62190002
                        at /home/tomi/projects/rust/rust-tensorflow/tensorflow-sys/build.rs:105
   8:     0x55d015df36b0 - build_script_build::main::h87abe698382f23c6
                        at /home/tomi/projects/rust/rust-tensorflow/tensorflow-sys/build.rs:78
   9:     0x55d015f5e04a - __rust_maybe_catch_panic
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libpanic_unwind/lib.rs:98
  10:     0x55d015f57876 - std::rt::lang_start::h65647f6e36cffdae
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:434
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panic.rs:351
                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/rt.rs:57
  11:     0x55d015df6aa2 - main
  12:     0x7f52f9b8a3f0 - __libc_start_main
  13:     0x55d015de3689 - _start
  14:                0x0 - <unknown>

```

**Comment by daschl** [comment]
@tmaila I already pushed a PR which bumps it to 1.0 TF so lets see if we can get this in sooner than later and resolve those issues. I ran into the same one at some point!

**Comment by adamcrume** [comment]
Fixed by #53.
